### DISCUSSION TITLE
/@id/__x00__plugin-vue:export-helper 404

### DISCUSSION THREAD

**Discussion by rch77** [vitejs/vite-plugin-vue#470] [General]
Hi there.
I'm getting 404 in browser for `GET https://******.com:5173/@id/__x00__plugin-vue:export-helper` and a page doesn't load, only the app template.
Then it shows:
`Uncaught (in promise) TypeError: error loading dynamically imported module: https://******.com:5173/resources/js/Pages/Forums.vue?t=1730821162019`

What I am missing?
The same vite 4 and laravel 10/inertia setup works fine when compiled for production or for dev with Svelte. `public/build/assets/_plugin-vue_export-helper-c27b6911.js` is built

**Comment by edison1105** [comment]
Please provide a minimal runnable reproduction

**Comment by rch77** [comment]
Thank you, I figured out the issue was with server:..proxy: clause in vite.config.js that was filtering `/@vite` & other requests with the rest going to the laravel dev server but missed this `/@id/` one.
